sdk/eventgrid/eventgrid/src/cloudEventDistrubtedTracingEnricherPolicy.ts

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Would love you feedback on this approach, Jeff.  We want the values we put in the event to be the same as what we put in the HTTP headers, and this seemed like the best way to do this.